### DISCUSSION TITLE
[Draft] Saucedacity/Tenacity 1.3 Release Agenda

### DISCUSSION THREAD

**Discussion by generic-pers0n** [saucedacity/saucedacity#62] [Announcements]
Having worked on a couple of things regarding merges and fixes, I've been thinking of a release agenda for the next release. Here's what's up.

# New
**From Audacity**:
* Clip titles (**in progress**; text rendering works)
* Context menus in the track area and for clips (**ready for merge**)

**From Tenacity**:
* New themes and cursors (**already done**)
* Dynamic compressor (**already done**)
* Major build system changes (merging Tenacity's build system)

# Changes
* Reintroduce a way to differentiate our preferences from Audacity's preferences to prevent confusion in our code between legacy versions of Audacity and modern versions of Saucedacity.

* More fixes (i.e., remove stop-gap solutions, namely `StackAllocator` which has a somewhat misleading name)

# Removals

_None at the moment_

---

This release agenda has not been finalized and is subject to change. I would like feedback on this release agenda to see where we can go for the next release.

**Comment by poyribn** [comment]
this is a great agenda because it's short.

especially with the Saucedacity→Tenacity rebranding and the major build system changes, having not much else to test may be needed for stability, especially since Saucedacity is a small project with (as far as i know) no paid (hired) developers yet.
